src/core/book-keeper.js

```diff
@@ -1,9 +1,277 @@
 'use strict';
 
-class BookKeeper {
-    constructor () {
+var Util = require('./util');
 
+var moment = require('moment');
+
+var metricsEventEmitter = require('../src/core/event/event-bus.js').EventEmitter;
+
+
+var init = function(EventI) {
+
+    EventI.on('DELETE_OLD_SCHEDULE', function(col, days, cb) {
```

Comment:
For this and all other events, create a constant in event.js for it and use the constant